rpcs3/rpcs3qt/tooltips.h

```diff
@@ -101,7 +101,7 @@ class Tooltips : public QObject
 		const QString debug_overlay                = tr("Provides a graphical overlay of various debugging information.\nIf unsure, don't use this option.");
 		const QString log_shader_programs          = tr("Dump game shaders to file. Only useful to developers.\nIf unsure, don't use this option.");
 		const QString disable_occlusion_queries    = tr("Disables running occlusion queries. Minor to moderate performance boost.\nMight introduce issues with broken occlusion e.g missing geometry and extreme pop-in.");
-		const QString force_cpu_blit_emulation     = tr("Forces emulation of all blit and image manipulation operations on the CPU.\nRequires 'Write Color Buffers' option to also be enabled in most cases to avoid missing graphics.\nSignificantly degrades performance but is more accurate in some cases.\nThis setting overrides the 'GPU texture scaling' option.");
+		const QString force_cpu_blit_emulation     = tr("Forces emulation of all blit and image manipulation operations on the CPU.\nRequires 'Write Color Buffers' option to also be enabled in most cases to avoid missing graphics.\nSignificantly degrades performance but is more accurate in some cases.\nThis setting overrides the 'GPU texture scaling' option.\nThis is also required for the waves to work proerly in VSH.");
```

Comment:
typo

Reply:
Including text explaining what game/app specific issues a setting fixed is dumb. It's only going to have to be changed again at a later date and there are too many games/apps that require a specific setting that only including one doesn't make much sense. It's already obvious that such a setting will help some games, otherwise why would it exist as an option